### ISSUE TITLE
Extend vtgate planbuilder to support other MySQL internal schema

### ISSUE THREAD

**Issue by giaquinti** [youtube/vitess#3061]
`vtgate` planbuilder supports queries to the MySQL `information_schema` by [routing them](https://github.com/youtube/vitess/blob/ac9f598b00bd635eaaa530059a936dc101e4bad3/go/vt/vtgate/planbuilder/from.go#L162-L168) to the default keyspace. 

What do we think about extending the logic to include as well other MySQL schemas like `sys` and `performance_schema`?

It could be nice to extend as well the support for `USE` statements. Right now they fail with an error:

```
mysql> use information_schema;
Database changed
mysql> show tables;
ERROR 1105 (HY000): vtgate: http://vitess:15001/: keyspace information_schema fetch error: node doesn't exist
```

**Comment by bbeaudreault** [comment]
I think this would be a relatively simple change. Can you PR?

My one concern is that `information_schema` tends to have data in it that is most likely applicable to all shards of a keyspace. Not sure about `sys`, but `performance_schema` probably has tablet-specific information. This isn't a blocker for me, but is something to be aware of.

**Comment by giaquinti** [comment]
I'm a golang newbie but I'll try to PR.

I agree with what you wrote but I think it can be anyway a useful feature especially if you only want to run a `SHOW`/`DESCRIBE` tables.

**Comment by sougou** [comment]
Addressed via #3109
